query/src/datasources/engines/metastore_clients/embedded_metastore.rs

```diff
@@ -95,6 +95,14 @@ impl MetaStoreClient for EmbeddedMetaStore {
                 let table = metas.name2meta.get(table_name).ok_or_else(|| {
                     ErrorCode::UnknownTable(format!("Unknown table: '{}'", table_name))
                 })?;
+
+                eprintln!(
```

Comment:
Perhaps we can use common trace for debug it :)

Reply:
`eprint`s will be removed in the final commit, it is used to fix some crash bugs to ensure that logs will be printed(without implicitly flush).

Thanks for reminding me